addons/hw_drivers/tools/helpers.py

```diff
@@ -439,12 +442,14 @@ def path_file(filename):
     elif platform_os == 'Windows':
         return Path().absolute().parent.joinpath('server/' + filename)
 
-def read_file_first_line(filename):
-    path = path_file(filename)
+
+def read_file_first_line(filename, absolute=False):
+    path = Path(filename) if absolute else file_path(filename)
```

Comment:
?

Reply:
`file_path` is not made for absolute paths